.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+Thanks for taking the time to contribute to the IBM Cloud docs! After you open your pull request, a maintainer should review it soon.
+
+We don't merge changes in this repository because content is not published from here. However, if we can use the changes that you suggest here, we publish them from our internal repository.
+
+### What does this pull request do?
```

Comment:
I'd lean toward something straightforward like "Pull request summary" since what it does is a very open-ended question. The helper text is great, but why make them read it 😉

Reply:
Used generic `Description` but kept helper text